app/controllers/references_controller.rb

```diff
@@ -11,7 +11,7 @@ def create
         flash['notice'] = 'This paper has already been added to this list'
       else
         Reference.create(list_id: list.id, paper_id: paper.id)
-        flash['notice'] = "You added #{paper.title} to #{list.name}"
+        flash['notice'] = "You added '#{paper.title}'' to '#{list.name}'"
```

Comment:
I think I have a similar change in my WIP branch. It's fine - I'll manage the merge when the conflict happens. However, you are erroneously using two apostrophes. at the end of `#{paper.title}`